Documentation/Diagnostics/PH2004.md

```diff
@@ -0,0 +1,44 @@
+# PH2004: Avoid ExpectedException Attribute
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MsTestAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MsTestAnalyzers) |
+| Diagnostic ID | PH2004 |
+| Category  | [Maintainability](../Maintainability.md) |
+| Analyzer | [ExpectedExceptionAttributeAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MsTestAnalyzers/ExpectedExceptionAttributeAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Error |
+| Enabled By Default | Yes |
+
+## Introduction
+
+Tests may not use the ExpectedException attribute. See (https://visualstudiomagazine.com/blogs/tool-tracker/2018/11/test-exceptions.aspx) for motivation.
```

Comment:
Give the link an name, using [name] syntax

Reply:
done